.flake8

```diff
@@ -35,7 +37,7 @@ per-file-ignores =
   # Unfortunately, flake8 does not allow to "noqa" just a specific error inside the file itself.
   # https://github.com/PyCQA/flake8/issues/1079
   #     F811 redefinition of unused '...'
-  stdlib/typing.pyi: B, E301, E302, E305, E501, E701, E741, NQA102, F401, F403, F405, F811, F822, Y037
+  stdlib/typing.pyi: B, E301, E302, E305, E501, E701, E741, NQA102, F401, F403, F405, F811, F822, Y037, Y041
```

Comment:
You'd need to add Y041 to lines 35 and 42 in this file, as well. Currently you're only ignoring Y041 in `stdlib/typing.pyi` specifically, not in any other stubs